packages/api/src/jobs/locales/i18n.js

```diff
@@ -4,7 +4,7 @@ import Backend from 'i18next-fs-backend';
 i18n.use(Backend).init(
   {
     fallbackLng: 'en',
-    preload: ['en', 'es', 'pt', 'fr'],
+    preload: ['en', 'es', 'pt', 'fr', 'de', 'hi', 'pa', 'ml'],
```

Comment:
Could we have an array of languages somewhere on the api side like you did on webapp, so that we don't have to alter that many files next time we add a new language?